src/server/frontend_xwayland/xwayland_wm.cpp

```diff
@@ -588,38 +605,27 @@ void mf::XWaylandWM::handle_configure_request(xcb_configure_request_event_t *eve
             log_warning("border width unsupported (border width %d)", event->border_width);
     }
 
-    auto shellSurface = surfaces[event->window];
-
-    uint32_t values[6];
-    int i = -1;
+    std::vector<uint32_t> values;
 
     if (event->value_mask & XCB_CONFIG_WINDOW_X)
-    {
-        values[++i] = event->x;
-    }
+        values.push_back(event->x);
```

Comment:
I mildly prefer always having `{ }` around the bodies of `if`, even if one-line.

I note that we don't have anything about this in the style guide. It's not a huge problem, though (gcc will warn if you do it wrong, mostly).